src/customStyle.js

```diff
@@ -21,12 +20,12 @@ export function isCustomStyleExists(styleName) {
 
 // [FS] IRAD-1128 2020-12-30
 // get a style by styleName
-export function getCustomStyleByName(name: string) {
-  let style = null;
+export function getCustomStyleByName(name: string): StyleProps {
```

Comment:
Inefficient.  If there are 10,000 styles and the first one is a match, this method will still loop over the remaining 9,999 styles.  Loop should end when style is found.

If called frequently enough, it would be even more efficient to store styles as a dictionary instead of an array.